test/ia-book-actions.test.js

```diff
@@ -421,7 +421,7 @@ describe('Auto renew one hour loan', () => {
     // timer-countdown reflects decreased loan time
     expect(el.timerCountdownEl.secondsLeftOnLoan).to.equal(8);
 
-    expect(el.postInitComplete).to.equal(33);
+    expect(el.postInitComplete).to.equal(false);
```

Comment:
q: why did we change this?

Reply:
its a boolean property which considered as false after during execution.

Reply:
Its a boolean property which expected to be false while execution.